packages/hoprd/crates/hoprd-hoprd/src/cli.rs

```diff
@@ -213,7 +212,7 @@ pub struct CliArgs {
         help = "Default channel strategy to use after node starts up",
         env = "HOPRD_DEFAULT_STRATEGY",
         value_name = "DEFAULT_STRATEGY",
-        value_parser = PossibleValuesParser::new([PromiscuousStrategy::NAME, PassiveStrategy::NAME ])
+        value_parser = PossibleValuesParser::new(["promiscuous", "passive" ])
```

Comment:
The previous approach was better... I suggest creating a full parser that would return the `Result<Strategy, ValidationError>` and convert it to Parser Result.

Reply:
I agree, please feel free to improve.